README.md

```diff
@@ -79,7 +79,7 @@ Once setup with a Go development environment and Docker, you can:
 3.  In another terminal window, switch to the directory and start an app:
 
         $ cd $GOPATH/src/github.com/openshift/origin
-        $ _output/local/go/bin/osc create -f examples/hello-openshift/hello-pod.json
+        $ _output/local/go/bin/osc create -f examples/hello-openshift/hello-pod.json --kubeconfig=$CERT_DIR/admin/.kubeconfig
```

Comment:
Do you mind moving this to its own line above as "export KUBECONFIG="?  I don't think CERT_DIR is guaranteed to be set either - if you do make run it would be the current directory so you can do `export KUBECONFIG=$(pwd)/openshift.local.certificates/admin/.kubeconfig`.  @deads2k, with `osc login` this should be easier in the future, is there something simpler we can do to make this easy for a new user?